main.go

```diff
@@ -1,55 +1,70 @@
 package main
 
 import (
+	"flag"
 	"os"
 
 	"github.com/pingcap/log"
 	"go.uber.org/zap"
 )
 
-var benchCases = map[string]func(*cluster) bench{
-	"scaleOut": newScaleOut,
-}
-
-func newBench(name string, c *cluster) bench {
-	if f, ok := benchCases[name]; ok {
-		return f(c)
-	}
-	return nil
-}
+var (
+	withExport = flag.Bool("export", false, "export mode, bench will creat data and export by br")
+	withBench  = flag.Bool("bench", true, "bench mode, it will bench this workload-scale-out")
+	withImport = flag.Bool("import", false, "import mode, it can be used with bench mode")
+	caseName   = flag.String("case", "", "case name, support list:scale-out,tpcc")
+)
 
 func main() {
+	flag.Parse()
+	// todo http head and format valid check
 	var clusterName = os.Getenv("CLUSTER_NAME")
 	var tidbServer = os.Getenv("TIDB_ADDR")
 	var pdServer = os.Getenv("PD_ADDR")
 	var prometheusServer = os.Getenv("PROM_ADDR")
 	var apiServer = os.Getenv("API_SERVER")
-	cluster := newCluster(clusterName, tidbServer, pdServer, prometheusServer, apiServer)
-	// load data
-	loader := newYcsb(cluster)
-	log.Info("load start")
-	err := loader.load()
-	if err != nil {
-		log.Fatal("failed when load", zap.Error(err))
+	// todo @zeyuan may additional parameters with export
+
+	cluster := NewCluster(clusterName, tidbServer, pdServer, prometheusServer, apiServer)
+	benchCases := NewBenches(cluster)
+	benchCase := benchCases.GetBench(*caseName)
+	if benchCase == nil {
+		log.Fatal("error with case name", zap.String("name", *caseName), zap.Strings("support list", benchCases.SupportList()))
+		return
 	}
-	log.Info("load finish")
 
-	// bench
-	bench := newBench("scaleOut", cluster)
-	if bench == nil {
-		log.Fatal("error bench name", zap.Error(err))
+	if *withExport {
+		err := benchCase.Import()
```

Comment:
Export？

Reply:
it's right, we need to import data firstly. I will add comment later